tests/ha/test_multi_server_pod_failure.py

```diff
@@ -1376,3 +1377,506 @@ def test_part_mpu_till_kserver_pods_fail(self):
+        assert_utils.assert_true(len(rem_bkts_aftr_del) < del_bucket,
+                                 "Some bucket deletion expected during pods going down")
+        LOGGER.info("Step 10.1: Verified status for In-flight DELETEs while %s (K) server"
+                    " pods were going down", self.kvalue)
+
+        LOGGER.info("Step 10.2: Verify status for In-flight WRITEs while %s (K) server pods going "
+                    "down should be failed/error.", self.kvalue)
+        responses_wr = dict()
+        while len(responses_wr) != 2:
+            responses_wr = output_wr.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_wr["pass_res"])
+        LOGGER.debug("Pass logs list: %s", pass_logs)
+        fail_logs = list(x[1] for x in responses_wr["fail_res"])
+        LOGGER.debug("Fail logs list: %s", fail_logs)
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), f"WRITEs logs which contain failures: {resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
+        assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
+                                 f"WRITEs logs which contain pass: {resp[1]}")
+        LOGGER.info("Step 10.2: Verified status for In-flight WRITEs while %s (K) server pods "
+                    "going down.", self.kvalue)
+
+        LOGGER.info("Step 10.3: Verify status for In-flight READs/Verify DI while %s (K)"
+                    " server pods going down should be failed/error.", self.kvalue)
+        responses_rd = dict()
+        while len(responses_rd) != 2:
+            responses_rd = output_rd.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_rd["pass_res"])
+        LOGGER.debug("Pass logs list: %s", pass_logs)
+        fail_logs = list(x[1] for x in responses_rd["fail_res"])
+        LOGGER.debug("Fail logs list: %s", fail_logs)
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), f"Reads logs which contain failures: {resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
+        assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
+                                 f"Reads logs which contain pass: {resp[1]}")
+        LOGGER.info("Step 10.3: Verified status for In-flight READs/Verify DI while %s (K)"
+                    " server pods going down.", self.kvalue)
+        LOGGER.info("Step 10: Verified status for In-flight READs/WRITEs/DELETEs while %s (K)"
+                    " server pods were going down.", self.kvalue)
+        LOGGER.info("ENDED: Test to verify continuous READs/WRITEs/DELETEs while %s (K) "
+                    "server pods were going down.", self.kvalue)
+
+    # pylint: disable=too-many-statements
+    # pylint: disable=too-many-locals
+    # pylint: disable=unused-argument
```

Comment:
This line we are not supposed to add. If there are any unused args please remove them and codacy error will go away.